openapi.yaml

```diff
@@ -0,0 +1,106 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Simple API
+  description: A simple API to illustrate OpenAPI concepts
+
+servers:
+  - url: https://example.io/v1
+
+
+paths:
+  /allFunds:
```

Comment:
Rename this to /allTransactions and add a new endpoint for retrieving the accounts, /allAccounts.